File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/INavigationItem.java
package org.openfoodfacts.scanner.utils;

/**
 * Created by Lobster on 06.03.18.
 */

import org.openfoodfacts.scanner.utils.NavigationDrawerListener.NavigationDrawerType;

public interface INavigationItem {

    NavigationDrawerListener getNavigationDrawerListener();

    @NavigationDrawerType
    int getNavigationDrawerType();

}

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/INavigationItem.kt
package openfoodfacts.github.scrachx.openfood.utils

import openfoodfacts.github.scrachx.openfood.utils.NavigationDrawerListener.NavigationDrawerType

/*
 Created by Lobster on 06.03.18.
*/
interface INavigationItem {
    val navigationDrawerListener: NavigationDrawerListener?

    @NavigationDrawerType
    fun getNavigationDrawerType(): Int
}